xml/System.Security.Cryptography.X509Certificates/CertificateRequest.xml

```diff
@@ -195,14 +264,27 @@
         <Parameter Name="serialNumber" Type="System.Byte[]" />
       </Parameters>
       <Docs>
-        <param name="issuerName">To be added.</param>
-        <param name="generator">To be added.</param>
-        <param name="notBefore">To be added.</param>
-        <param name="notAfter">To be added.</param>
-        <param name="serialNumber">To be added.</param>
-        <summary>To be added.</summary>
-        <returns>To be added.</returns>
+        <param name="issuerName">The X500DistinguishedName for the Issuer.</param>
+        <param name="generator">An <see cref="T:System.Security.Cryptography.X509Certificates.X509SignatureGenerator"/> object representing the issuing certificate authority.</param>
+        <param name="notBefore">The oldest date and time when this certificate is considered valid. Typically <see cref="P:System.DateTimeOffset.UtcNow"/>, plus or minus a few seconds.</param>
+        <param name="notAfter">The date and time when this certificate is no longer considered valid.</param>
+        <param name="serialNumber">The serial number to use for the new certificate. This value should be unique per issuer. The value is interpreted as an unsigned (big) integer in big-endian byte ordering.</param>
```

Comment:
Here and in the previous one, I'm wondering if my "unsigned (big) integer" is a weird construction.  It certainly made sense to me when I wrote it, but without coffee it felt odd.

The intent was "an unsigned integral value, but not confined to 4 bytes like the 'int' type".  So maybe "an unsigned integral value", or "an unsigned integer of arbitrary size".

It might also be worth noting somewhere that RFC 3280 (https://tools.ietf.org/html/rfc3280#section-4.1.2.2) recommends confining it to 20 bytes or less.